website/content/docs/nia/enterprise/license.mdx

```diff
@@ -27,13 +27,13 @@ Choose one of the following methods (in order of precedence) to set the license:
 export CONSUL_LICENSE=<LICENSE_VALUE>
 ```
 
-2. Set the `CONSUL_LICENSE_PATH` environment variable to the path of the file containing the license.
+1. Set the `CONSUL_LICENSE_PATH` environment variable to the path of the file containing the license.
 
 ```shell-session
 export CONSUL_LICENSE_PATH=<PATH>/<TO>/<FILE>
 ```
 
```

Comment:
I believe the numbering issues would be resolved if the code blocks are indented.
```suggestion
1. Set the `CONSUL_LICENSE_PATH` environment variable to the path of the file containing the license.

    ```shell-session
    export CONSUL_LICENSE_PATH=<PATH>/<TO>/<FILE>
    ```

```

Reply:
@blake I just tried that and the extra indentation has no effect as it's already indented from the bullet anchor (see image). The bullets should also be all `1.` per markdown syntax. 

![image](https://user-images.githubusercontent.com/29551334/131885318-fe69434f-fc10-4514-8cd7-fe51702b0898.png)